### ISSUE TITLE
Formatting of signed content

### ISSUE THREAD

**Issue by woods1984** [MrMarchello/php-XmlDigitalSignature#3]
Thanks for this library and excellent documentation.

I've got it up and running and producing a valid signed xml doc.  However, my content (not the signature) seems to be encoded to UTF-8 text instead of xml.  I don't understand this stuff too well, so I may be describing this wrong, but what I mean is that the xml comes out like this...

```<Object Id="licence">&lt;licence activationCode="1234-1234-1234-1234" createdOn="2015-11-29 20:18:32" &lt;/licence&gt;</Object>```

Whereas I was expecting

```<Object Id="licence"><licence activationCode="1234-1234-1234-1234" createdOn="2015-11-29 20:18:32" </licence></Object>```

Is this normal?  If so, is there a way to prevent this from happening?

The code I've used is below
```php
 // If a licence was found then get the licence file
  $xml = new DOMDocument('1.0', 'UTF-8');
  $xmlLicence=$xml->createElement("licence");
  $xmlLicence=$xml->appendChild($xmlLicence);
  $xmlLicence->setAttribute("guid", "{30397799-EA0A-569A-2DA5-FC3EB3B400EA}");
  // More attributes removed
  
  // Gets a signature to add to the xml
  require(__DIR__ . "/php-XmlDigitalSignature-master/src/XmlDigitalSignature.php");
  $dsig = new XmlDsig\XmlDigitalSignature();
  $dsig->setNodeNsPrefix('');
  $dsig
	->setCryptoAlgorithm(XmlDsig\XmlDigitalSignature::RSA_ALGORITHM)
	->setDigestMethod(XmlDsig\XmlDigitalSignature::DIGEST_SHA1)
	->setCanonicalMethod(XmlDsig\XmlDigitalSignature::C14N_EXCLUSIVE)
	->forceStandalone();
	
  // Load the keys	
  try
  {
	$dsig->loadPrivateKey(__DIR__ . '/privateLicenceKey.pem');		
	$dsig->loadPublicKey(__DIR__ . '/publicLicenceKey.pem');
	$dsig->loadPublicXmlKey(__DIR__ . '/publicLicenceKey.xml');
  }
  catch (\UnexpectedValueException $e)
  {
    // Display error (removed)
  }
 
  // Sign and verify the signature
  try
  {
	$dsig->addObject($xmlLicence, 'licence');
	$dsig->sign();
	$dsig->verify();
  }
  catch (\UnexpectedValueException $e)
  {
	// Display error (removed)
  }
  $licenceString=$dsig->getSignedDocument();
```

**Comment by MrMarchello** [comment]
Can you please post the code and full XML output (including expected output) here using markdown? It's very difficult to make sense of it as it is.

**Comment by woods1984** [comment]
Sorry about that!  My first post on github and the markdown took me by suprise, but I've sorted it now

**Comment by MrMarchello** [comment]
Hi, apologies for the belated response. This has been fixed in `3aa643e`, please confirm and I will close this issue.

**Comment by woods1984** [comment]
Hi.  Thanks a lot for fixing that - I've just checked and it works fine now.
